packages/itwin/measure-tools/src/tools/MeasureAreaTool.ts

```diff
@@ -28,6 +28,7 @@ import { MeasureTools } from "../MeasureTools";
 import { SheetMeasurementsHelper } from "../api/SheetMeasurementHelper";
 import type { DrawingMetadata } from "../api/Measurement";
 import { DrawingDataCache } from "../api/DrawingTypeDataCache";
+import { ViewHelper } from "../measure-tools-react";
```

Comment:
Barrel import -- please fix.
edit: other files import from the barrel file as well.